mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -112,25 +112,24 @@ def _init_tables_path(self):
         ) = mlrun.common.model_monitoring.helpers.parse_model_endpoint_store_prefix(
             monitoring_predictions_full_path
         )
-        self.tables[mm_constants.FileTargetKind.PREDICTIONS] = (
-            monitoring_predictions_path
-        )
+        self.tables[mm_schemas.FileTargetKind.PREDICTIONS] = monitoring_predictions_path
 
-    def create_tsdb_application_tables(self):
+    def create_tsdb_application_tables(self) -> None:
         """
         Create the application tables using the TSDB connector. At the moment we support 2 types of application tables:
         - app_results: a detailed result that includes status, kind, extra data, etc.
         - metrics: a basic key value that represents a single numeric metric.
         """
         application_tables = [
-            mm_constants.MonitoringTSDBTables.APP_RESULTS,
-            mm_constants.MonitoringTSDBTables.METRICS,
+            mm_schemas.MonitoringTSDBTables.APP_RESULTS,
+            mm_schemas.MonitoringTSDBTables.METRICS,
         ]
-        for table in application_tables:
-            logger.info("Creating table in V3IO TSDB", table=table)
+        for table_name in application_tables:
+            logger.info("Creating table in V3IO TSDB", table_name=table_name)
+            table = self.tables[table_name]
             self._frames_client.create(
                 backend=_TSDB_BE,
-                table=self.tables[table],
```

Comment:
Can you explain why is better to pass a different "new" variable in this case?

Reply:
It's the same, but I think that the previous version is more confusing compared to
```py
table = self.tables[table_name]
```